### ISSUE TITLE
This extension isn't working with Python 3

### ISSUE THREAD

**Issue by oz123** [tonybaloney/sphinxcontrib-confluencebuilder#3]
> The xmlrpclib module has been renamed to xmlrpc.client in Python 3. The 2to3 tool will automatically adapt imports when converting your sources to Python 3. 
```

  $ sphinx-build -b confluence -d _build/doctrees . _build/html -E -a
Running Sphinx v1.4.8

Extension error:
Could not import extension sphinxcontrib.confluencebuilder (exception: No module named 'xmlrpclib')
```

**Comment by oz123** [comment]
Update - I think only the version in pypi is broken ...

**Comment by tonybaloney** [comment]
Thanks. I'll use six.moves and republish. 

Anthony Shaw




On Fri, Jan 20, 2017 at 7:27 PM +1100, "Oz N Tiram" <notifications@github.com> wrote:












Update - I think only the version in pypi is broken ...

**Comment by tonybaloney** [comment]
confirmed fixes https://travis-ci.org/tonybaloney/sphinxcontrib-confluencebuilder/jobs/194315238

**Comment by tonybaloney** [comment]
released 0.3.0 to PyPI. please upgrade
